lib/cocoapods/target/aggregate_target.rb

```diff
@@ -7,6 +7,9 @@ class AggregateTarget < Target
     #         generated this target.
     attr_reader :target_definition
 
+    # Product types where the product's frameworks must be embedded in a host target
+    EMBED_FRAMEWORKS_IN_HOST_TARGET_TYPES = [:app_extension, :watch_extension].freeze
```

Comment:
then it raises if it doesn't, but for submitting to the app store it will need to embed in frameworks host targets